### PR TITLE
Switched to using service region field

### PR DESCRIPTION
**Pull request by Edzelopez**

### CODE REVIEW

**Comment by seamuslee001** [comment]
so @Edzelopez we would need to have deployed that other Job right first?

**Comment by Edzelopez** [comment]
Yes, that is right. On staging, that job has been run, so maybe you can try testing this there?

**Comment by seamuslee001** [comment]
also the following files / lines of code either reference directly the custom field affected here or reference the term Region so need to be reviewed for impacts

https://github.com/JMAConsulting/biz.jmaconsulting.ao/blob/master/ROCR.php#L80
https://github.com/JMAConsulting/biz.jmaconsulting.aoreports/blob/master/CRM/AOReports/Form/Report/ServiceNavigation.php
https://github.com/JMAConsulting/biz.jmaconsulting.aoreports/blob/master/CRM/AOReports/Form/Report/FamiliesServedByRegion.php
https://github.com/JMAConsulting/biz.jmaconsulting.aoreports/blob/master/CRM/AOReports/Form/Report/SNPWAITDAY.php
https://github.com/JMAConsulting/biz.jmaconsulting.aoreports/blob/master/CRM/AOReports/Form/Report/MinistryRegion.php
https://github.com/JMAConsulting/biz.jmaconsulting.aoreports/blob/master/CRM/AOReports/Form/Report/ServiceNavigationDetail.php
https://github.com/JMAConsulting/biz.jmaconsulting.aoservicenav/blob/master/aoservicenav.php#L281
https://github.com/JMAConsulting/biz.jmaconsulting.aoservicenav/blob/master/aoservicenav.php#L469

**Comment by Edzelopez** [comment]
I've added in a couple of PRs to address the files above:

1. ROCR.php does not need any updates, it was a one time migration script we haven't used since.

https://github.com/JMAConsulting/biz.jmaconsulting.aoreports/pull/24

https://github.com/JMAConsulting/biz.jmaconsulting.aoservicenav/pull/1

File: CRM/CU/Upgrader.php [hunk 1]
```diff
@@ -68,6 +68,16 @@ public function upgrade_4200() {
     return TRUE;
   } // */
 
+  public function upgrade_1100() {
+    $this->ctx->log->info('Applying update 1100 - Delete region custom field');
+    $regionId = civicrm_api3('CustomField', 'getvalue', array(
```
Comment (seamuslee001) [inline CRM/CU/Upgrader.php:72]: @Edzelopez this is still deleting the field I thought we we were going to disable it

Comment (Edzelopez) [reply CRM/CU/Upgrader.php:72]: Fixed

**Pull request merged**